projects/packages/forms/package.json

```diff
@@ -33,6 +33,7 @@
 	"dependencies": {
 		"@automattic/jetpack-analytics": "workspace:*",
 		"@automattic/jetpack-components": "workspace:*",
+		"@automattic/jetpack-script-data": "workspace:*",
```

Comment:
This is added, based on feedback, so we can use the [getAdminUrl](https://github.com/Automattic/jetpack/blob/8e30f68bad9351aafefec7d545fb9ee62d8c84d0/projects/js-packages/script-data/src/utils.ts#L28) method.